crates/types/src/services/txpool.rs

```diff
@@ -306,7 +307,19 @@ pub enum Error {
     TTLReason,
     #[error("Transaction squeezed out because {0}")]
     SqueezedOut(String),
+    #[error("Invalid transaction data: {0:?}")]
+    ConsensusValidity(CheckError),
+    #[error("Mint transactions are disallowed from the txpool")]
+    MintIsDisallowed,
+    #[error("Database error: {0}")]
+    Database(String),
```

Comment:
It seems you can remove `Other` variant=)

Reply:
It's used in a couple edge cases still, since we didn't have anyhow to lean on anymore.